app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/organisation/GrsController.scala

```diff
@@ -116,7 +120,12 @@ class GrsController @Inject() (
     registration: Registration
   )(implicit hc: HeaderCarrier, request: JourneyRequest[AnyContent]): Future[RegistrationStatus] =
     if (registration.organisationDetails.businessVerificationFailed)
```

Comment:
businessVerificationFailed means the business verfication step of the GRS journey failed NOT GRS failed to verify a company. This word is therefore still applicable for Sole traders who can also go down the business verification step.